docs/Reference/Request.md

```diff
@@ -42,6 +42,8 @@ Request is a core Fastify object containing the following fields:
   handling the request
 - `routeConfig` - The route [`config`](./Routes.md#routes-config) 
   object.
+- `routeOptions`
+  - `bodyLimit` - either server limit or route limit
```

Comment:
Reference links to the full documentation for this option would be very helpful.

Reply:
done 👍

Reply:
I do not see a link to the body limit docs.

Reply:
Unfortunately, it didn't stage but now it's there.